pkg/util/util.go

```diff
@@ -1173,6 +1176,21 @@ func GetAWSRegion() (string, error) {
 		return "", fmt.Errorf("Unexpected node name format: %q", nodesList.Items[0].Name)
 	}
 	awsRegion := mapValidAWSRegions[nameSplit[1]]
+
+	// Fetching aws region from cluster if not fetched from node name
+	if awsRegion == "" {
+		infrastructure := &configv1.Infrastructure{}
+		infrastructure.SetName("cluster")
```

Comment:
I think this should work:

```suggestion
		infrastructure := &configv1.Infrastructure{Name: "cluster"}
```

Reply:
Updated, Thanks